src/event-handler/index.ts

```diff
@@ -15,12 +15,18 @@ import {
 import { receiverHandle as receive } from "./receive.js";
 import { removeListener } from "./remove-listener.js";
 
-interface EventHandler<TTransformed> {
+interface EventHandler<TTransformed = unknown> {
```

Comment:
Just wondering if it's necessary for this value to be optional? As this interface is never called on it's own, and `TTransformed` is always passed down

Reply:
i think it is an artifact from my testing
```suggestion
interface EventHandler<TTransformed> {
```